shared/chat/conversation/messages/account-payment/container.js

```diff
@@ -77,8 +79,8 @@ const mapStateToProps = (state, ownProps: OwnProps) => {
           paymentInfo.amountDescription
         )}`,
         balanceChangeColor: WalletConstants.balanceChangeColor(paymentInfo.delta, paymentInfo.status),
-        cancelButtonInfo: youAreSender && cancelable ? makeCancelButtonInfo(theirUsername) : '',
-        cancelButtonLabel: youAreSender && cancelable ? 'Cancel' : '',
+        cancelButtonInfo: paymentInfo.showCancel ? makeCancelButtonInfo(theirUsername) : '',
```

Comment:
there are a couple of other places where we condition on 'cancelable', e.g. shared/chat/conversation/messages/message-popup/payment/container.js . Do we still need 'cancelable' to be a possibility in `statusSimplified`? Can't this new `showCancel` field replace that?

Reply:
I think I see what you're saying about message-popup. Not sure how to get message-popup to show up in this state.

Reply:
per our chat, I think the case in payment/container.js is fine as-is (or will be once the status is renamed to be `claimable`) since it's a popup for the sender